custom_components/opnsense/queue.py

```
@@ -0,0 +1,22 @@
+queue: asyncio.Queue = asyncio.Queue(maxsize=1)
+
+
+async def handle_queue():
+    """Asyncio queue handler."""
+    while True:
+        try:
+            func = await queue.get()
+            if callable(func):
+                await func()
+            queue.task_done()
+        except Exception as e:  # noqa: BLE001
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)